plugin.json

```diff
@@ -105,5 +105,50 @@
             }
         ],
         "footer": "* To report an issue, make a suggestion or a contribution, [check the repository](https://github.com/mattermost/mattermost-plugin-github)."
+    },
+    "props":{
```

Comment:
this is a branch created from issue  #379  , which is created for  adding variables to mainfest.go

Reply:
If the change isn't related to this PR, the change should be removed from this PR

Reply:
I haven't ever seen this pattern either. This is to share values between the server and webapp portions of the plugin?

Reply:
To minimize the diff between this PR and the other PR that is in progress, you may be able to have this PR's base branch set to the other PR's branch. Though it may not let you do this because it's on a fork

Reply:
yes have tried creating base branch as the other one but it only let us to create PR with MM branch

Reply:
> I haven't ever seen this pattern either. This is to share values between the server and webapp portions of the plugin?

yes in the #379  this was a suggestion to use by @hanzei  [here](https://github.com/mattermost/mattermost-plugin-github/pull/456#discussion_r670259213) 
after figuring out is there any way , i felt props is the only where we can add new values as rest is with static structure

Reply:
> yes have tried creating base branch as the other one but it only let us to create PR with MM branch

@sibasankarnayak Are you able to create branches in Mattermost's repo with your current permissions? cc @hanzei

Reply:
@mickmister No tried creating a branch and push but unable to do so
cc @hanzei 
![Screenshot 2021-09-10 at 7 48 02 AM](https://user-images.githubusercontent.com/86650698/132788739-91c484f8-bb92-4871-8088-cfe88853dce0.png)